hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -822,7 +836,7 @@ public void testLookupKeyWithLocation() throws IOException {
   @Test
   public void testLatestLocationVersion() throws IOException {
     String keyName = RandomStringUtils.randomAlphabetic(5);
-    OmKeyArgs keyArgs = createBuilder()
+    OmKeyArgs keyArgs = createBuilder(VERSIONED_BUCKET_NAME)
```

Comment:
this test was failing because it expects the old "versioned" style of bucket, which is no longer turned on by default.  So fixed this test to turn it on.  (I also had to mock the Pipeline so that the request would succeed.)